ocaml/lib/pickles_bindings.ml

```diff
@@ -298,9 +308,14 @@ module Choices = struct
           |> Promise_js_helpers.of_js
           |> Promise.map ~f:(finish_circuit prevs self)
         in
-
         { identifier = Js.to_string rule##.identifier
-        ; feature_flags = rule##.featureFlags
+        ; feature_flags =
```

Comment:
I changed the type of the pickles rule here as well to match the rest (less different types we have to consinder in javascript or here)